controller/workitem_blackbox_test.go

```diff
@@ -2481,3 +2583,40 @@ func (s *workItemChildSuite) TestWorkItemListFilterByNoParents() {
 	})
 
 }
+
+func createSpaceWithDefaults(ctx context.Context, db *gorm.DB) (*space.Space, *iteration.Iteration, *area.Area) {
+	spaceRepo := space.NewRepository(db)
+	iterationRepo := iteration.NewIterationRepository(db)
+	areaRepo := area.NewAreaRepository(db)
+
+	newSpace := space.Space{
+		Name: fmt.Sprintf("The Space %v", uuid.NewV4()),
+	}
+	sp, err := spaceRepo.Create(ctx, &newSpace)
+	if err != nil {
+		fmt.Println("Failed to create space for area.")
```

Comment:
yes yes.. mistaken: done in https://github.com/almighty/almighty-core/pull/1241/commits/b99ed72c22013187555d82b436fbd907ab783ffb